logger/logger.go

```diff
@@ -50,21 +113,31 @@ func CreateLogFile(dirPath, filename string) (io.WriteCloser, error) {
 	return file, nil
 }
 
-// SetInfos set logger information from arguments
-func (l *Logger) SetInfos(file io.WriteCloser, flag int, debug bool) {
-	l.File = file
-	if debug {
-		l.Debug = log.New(io.MultiWriter(os.Stderr, file), "", flag)
-	} else {
-		l.Debug = log.New(file, "", flag)
+func (l *Logger) buildDebugLogger() {
+	if l.isDebug && l.file != nil {
```

Comment:
If the file exists only when `--debug` is used, I think then one of the conditions should be enough.

Reply:
Exactly, I already forget to set isDebug = true.